src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -0,0 +1,169 @@
+import { clone, extend, reduce, keys } from 'underscore';
+import dayjs from 'dayjs';
+import store from 'store';
+
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+export default Backbone.Model.extend({
+  defaults() {
+    return {
+      actionsSortId: 'sortCreatedDesc',
+      flowsSortId: 'sortCreatedDesc',
+      filters: {
+        type: 'flows',
+        groupId: null,
+        clinicianId: this.currentClinician.id,
+        roleId: this.currentClinician.getRole().id,
+        dateType: 'created_at',
+        selectedDate: null,
+        selectedMonth: null,
+        relativeDate: null,
+      },
+      selectedActions: {},
+      selectedFlows: {},
+    };
+  },
+  preinitialize() {
+    this.currentClinician = Radio.request('bootstrap', 'currentUser');
+    this.groups = this.currentClinician.getGroups();
+  },
+  initialize() {
+    this.on('change', this.onChange);
+  },
+  onChange() {
+    store.set(`${ this.id }_${ this.currentClinician.id }`, this.attributes);
+  },
+  getFilters() {
+    return clone(this.get('filters'));
+  },
+  getType() {
+    return this.getFilters().type;
+  },
+  isFlowType() {
+    return this.getType() === 'flows';
+  },
+  getSort() {
+    return this.get(`${ this.getType() }SortId`);
+  },
+  setSort(sortId) {
+    this.set(`${ this.getType() }SortId`, sortId);
+  },
+  getEntityDateFilter() {
+    // NOTE: Default dateType handles missing filter in current localStorage
```

Comment:
Added a note here, but because filters is nested on the state, it gets cleared by anything in the localstorage.  So far this isn't a problem for the other state since it's all null, but `dateType` being null here was a problem.  @LisaManresa you may need to do the same.  I'm not sure though.. maybe we should be extending the localstorage filter with the defaults so this kind of thing isn't necessary?

Reply:
I think I would lean toward extending localStorage with the defaults. Especially if we're going to have to make this accommodation in more than one place.